### ISSUE TITLE
schedule_viewer service period display and selection

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SEBDYA/googletransitdatafeed#17]
```
schedule_viewer should provide a way to browse service periods, display
them on a calendar and allow the user to select periods.

Currently the list of departure times is a little confusing because it
displays times in any service period instead of the service on a particular
day. 
```

Original issue reported on code.google.com by `tom.brow...@gmail.com` on 28 Apr 2007 at 3:21

**Comment by GoogleCodeExporter** [comment]
Original comment by `tom.brow...@gmail.com` on 30 Apr 2007 at 3:19
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
The current display isn't great. It displays all trips from all service periods 
as
though they run in parallel on the same day. And there is no filter to select a
calendar date, day of week or service period. If someone else puts together a 
UI to
select some of these I'll make the changes to make it work.
```

Original comment by `tom.brow...@gmail.com` on 15 May 2007 at 5:52

**Comment by GoogleCodeExporter** [comment]
```
+1 vote for addressing this feature request
```

Original comment by `bdfer...@gmail.com` on 7 Oct 2008 at 8:05

**Comment by GoogleCodeExporter** [comment]
```
Version 1.1.6 made a slight improvement, next to each trip in the stop bubble 
is the
service_id.
```

Original comment by `tom.brow...@gmail.com` on 7 Oct 2008 at 11:24

**Comment by GoogleCodeExporter** [comment]
```
This would be a very useful enhancement for us at Metro North, as we would like 
to
test that the disabling of certain service dates via the calendar_dates file is
working properly.
```

Original comment by `davelon...@gmail.com` on 25 Nov 2008 at 11:23

**Comment by GoogleCodeExporter** [comment]
```
All yours Andre!
```

Original comment by `tom.brow...@gmail.com` on 16 Nov 2009 at 7:26

**Comment by GoogleCodeExporter** [comment]
```
Partially addressed by r1300, which adds single date selection. The next steps 
would 
probably be to add date range selection, and more info about the dates in which 
the 
selected service runs.

It would also be nice to refresh the trip list when a new date is selected.
```

Original comment by `a...@google.com` on 7 May 2010 at 12:54

**Comment by GoogleCodeExporter** [comment]
```
There is a slight issue with the calendar selection in schedule_viewer.  For 
me, when using either Chrome or IE, selecting a date in November or December 
places a 4-digit number in the Date box instead of the date in the format 
yyyyMMDD.  
```

Original comment by `devin.br...@gmail.com` on 17 Sep 2010 at 11:11

**Comment by GoogleCodeExporter** [comment]
Original comment by `christin...@gmail.com` on 2 Aug 2014 at 5:15

Attachments:
- [image.jpg](https://storage.googleapis.com/google-code-attachments/googletransitdatafeed/issue-17/comment-9/image.jpg)

**Comment by GoogleCodeExporter** [comment]
```
Moved to https://github.com/google/transitfeed/issues/17
```

Original comment by `bdfer...@google.com` on 7 Oct 2014 at 7:52
- Changed state: **WontFix**
